jsonrpc/PersistentStore.json

```diff
@@ -0,0 +1,480 @@
+{
```

Comment:
Although the json files are of course still supported was it perhaps considered to add the json interface to the C++ header file using the meta tags? (e.g. see https://rdkcentral.github.io/Thunder/plugin/interfaces/interfaces/ "Enable JSON-RPC Generation" and here for tags that enable you to influence the generated json rpc interface to make it backward compatible if necessary: https://rdkcentral.github.io/Thunder/plugin/interfaces/tags/)

Again the json file is fine but perhaps it is goof to be aware it is also possible to define the com rpc and json rpc interface in one file (and therefore guarantee they are consistent and kept in sync). If there were features missing that made it impossible to use the above please let us know, we are always happy to learn about it and see if we can extend the support for this.

Reply:
@npoltorapavlo - Pls check and respond here. If it's lot of work we can merge these changes now and make subsequent changes to move to using the meta tags.